Cargo.toml

```diff
@@ -159,12 +156,16 @@ tracing-subscriber = { version = "0.3.18", default-features = false, features =
   "tracing-log",
 ] }
 triomphe = { version = "0.1.14", default-features = false, features = ["std"] }
-url = "2.3.1"
-xshell = "0.2.5"
+url = "2.5.4"
+xshell = "0.2.7"
 
 
 # We need to freeze the version of the crate, as the raw-api feature is considered unstable
 dashmap = { version = "=5.5.3", features = ["raw-api"] }
```

Comment:
It'd be good to bump dashmap to `7.0.0-rc2` (here, and in Salsa), as it'd allow us to get rid of the `raw-api` feature entirely: it's replaced by hashbrown's newly-stable `HashTable`.

Unfortunately, when I tried doing so, the changes to the interning macro in `hir` was kinda difficult  to wrap my head around and I meant to open an issue to ask for help/clarification, so... I did it on this PR.

Reply:
Let's do that separately

Reply:
Looking into this, what makes you think we can drop the `raw-api` feature? We still need that